src/main/java/net/mcreator/blockly/data/Dependency.java

```diff
@@ -132,7 +131,7 @@ public String getDependencyBlockXml() {
 	public static Color getColor(String type) {
 		// Check if the type is a loaded variable and then, get its HUE color
 		if (VariableTypeLoader.INSTANCE.fromName(type) != null) {
-			return BlocklyBlockUtil.getBlockColorFromHUE(VariableTypeLoader.INSTANCE.fromName(type).getColor());

```

Comment:
Do we still use BlocklyBlockUtil.getBlockColorFromHUE? If not, you should remove it.

Reply:
It's still used by procedure categories and procedure element preview